tests/examples/test_gcd.py

```diff
@@ -43,11 +43,11 @@ def test_py(setup_example_test):
     # "Creating fake entries in the LUT"
     assert chip.get('metric', 'warnings', step='cts', index='0') == 2
 
-    # Missing route to pin (x77)
-    assert chip.get('metric', 'warnings', step='route', index='0') == 77
+    # Missing route to pin (x71)
+    assert chip.get('metric', 'warnings', step='route', index='0') == 71
```

Comment:
Good to see the number of warnings going down, but I've also had a recent change get tripped up on this test's number-of-warnings checks. Should we make a note to update this test to be less rigid, maybe looking for a list of expected error classes instead of an exact number?

Reply:
This test is way too sensitive, these are a "bug" in OpenROAD. Them going up or down doesn't mean a lot for our flow implementation.